packages/elements/src/slider/__test__/slider.event.test.js

```diff
@@ -1273,4 +1273,62 @@ describe('slider/Events', function () {
     // Check call fire event
     expect(callCountValue).to.equal(1);
   });
+
+  it('Should fires input event when dragging thumb slider on desktop', async function () {
```

Comment:
does this test cover touch devices? We need to test them as well.

Reply:
these thumb slider test cases are not cover the touch devices (do we have touch devices test case? I can't remember that). If we need to cover it, I think we'll have a lot of thing to do.
should we create a new ticket for this instead?

Reply:
mobile user agents might handle mouse related event as touch events automatically. Let me check.